### ISSUE TITLE
Process explorer should remember it's location and dimensions

### ISSUE THREAD

**Issue by Tyriar** [microsoft/vscode#119458]
It seems to always launch in the center of the active window, so whenever I'm open it I need to drag it out to the other monitor.

**Comment by weartist** [comment]
Do this request still accept PR? I would like to try it, we can record the last position and size and create it with this value the next time we open it.
and I found that the current code expects the same window as vscode every time it starts, If needed, we can add a switch to control whether to follow the parent view's window or not? 


https://github.com/microsoft/vscode/blob/dd93163b73418600c48ede98aa78b072e7a0cde4/src/vs/platform/issue/electron-main/issueMainService.ts#L352

**Comment by Tyriar** [comment]
@weartist sounds good 👍, I think just  a single global position is probably the best as the process explorer is shared across all windows.

**Comment by weartist** [comment]
> @weartist sounds good 👍, I think just  a single global position is probably the best as the process explorer is shared across all windows.

let me gave it a try

**Comment by jrieken** [comment]
Position is good but size seems pretty random

https://user-images.githubusercontent.com/1794099/214634753-b1f0e2e9-0d5c-406d-a374-e59770a0ba58.mov

**Comment by Tyriar** [comment]
@jrieken do you have multiple monitors? What's the devicePixelRatio of your monitors?

**Comment by jrieken** [comment]
yes, multi-monitor and no clue what devicePixelRatio is. Is that zoom related or how do I find that out?

**Comment by Tyriar** [comment]
@jrieken yeah that's the scaling factor of the window, open devtools and run `window.devicePixelRatio` when the window is on each monitor.

**Comment by jrieken** [comment]
Thanks for the intro - it prints `2` on both screens

Also looping in @bpasero since this is working fine for normal code windows

**Comment by bpasero** [comment]
Hm, not sure there is something specific I do. Getting the bounds for persistence is done here:

https://github.com/microsoft/vscode/blob/0785b1c28204ca0f0584fb2fe8810142ab989069/src/vs/platform/windows/electron-main/windowImpl.ts#L1125

And restoring here:

https://github.com/microsoft/vscode/blob/0785b1c28204ca0f0584fb2fe8810142ab989069/src/vs/platform/windows/electron-main/windowImpl.ts#L1219

**Comment by Tyriar** [comment]
@bpasero suggested to share the logic for the regular Windows which is probably the fix here, I didn't want to invest much effort into this though.

**Comment by weartist** [comment]
@jrieken Hi, can you still reproduce the problem, I tested it with multiple monitors and did not find the size problem, slittle strange

**Comment by legomushroom** [comment]
Was able to catch another important issue - if 2 monitors are used on windows, and then the second monitor disconnected, the explorer window gets inaccessible as if it is "stuck" on the disconnected monitor. 

Repro steps:

- use windows machine and a laptop with an external monitor
- open vscode workbench
- open process explorer and drag the window to the second monitor
- close the process explorer
- disconnect the second monitor, open the process explorer again
- ⚠ notice that the process explorer window is not accessible as if it is left on the other monitor, the only way to get it back is to connect the external monitor again; this issue is persists even after an OS reboot

**Comment by bpasero** [comment]
This is now dealt with according to floating window rules, as the process explorer has moved to be a floating window component.
